meta/runtime.yml

```diff
@@ -1,5 +1,39 @@
 requires_ansible: ">=2.13.0"
 
+action_groups:
+  all:
```

Comment:
Surely no one will ever forget to add a new resource to this list :upside_down_face: